app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,61 @@
+class ReviewsController < ApplicationController
+  def new
+    @restaurant= Restaurant.find(params[:restaurant_id])
+    @review = Review.new
+  end
+
+  def create
+    @restaurant= Restaurant.find(params[:restaurant_id])
+    @review = @restaurant.reviews.new(review_params)
+    @review.user = current_user
+    if @review.save
+      flash[:notice] = "Review Created!"
+      redirect_to restaurant_path(@restaurant)
+    else
+      render :new
+    end
+  end
+
+    def destroy
```

Comment:
Assignment Branch Condition size for destroy is too high. [18.14/15]<br>Inconsistent indentation detected.